[BUG] Helm installation is failing with ATP

**Describe the bug**
error during helm installation

```
 Error: INSTALLATION FAILED: template: rabbitmq-chart/templates/integration_tests/deployment.yaml:49:40: executing "rabbitmq-chart/templates/integration_tests/deployment.yaml" at <include (print $.Template.BasePath "/atp-storage-secret.yaml") .>: error calling include: template: no template "rabbitmq-chart/templates/atp-storage-secret.yaml" associated with template "gotpl"
  Error: Process completed with exit code 1.
```

**To Reproduce**
Steps to reproduce the behavior:
Run test pipeline
Job with ATP integration enabled fail
https://github.com/Netcracker/qubership-rabbitmq/actions/runs/31350804832/job/93341137721

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Environment:**
 - Application Version: 
 - K8S Version: 

**Additional context**
Add any other context about the problem here.

--- a/operator/charts/helm/rabbitmq/templates/integration_tests/deployment.yaml
+++ b/operator/charts/helm/rabbitmq/templates/integration_tests/deployment.yaml
@@ -46,7 +46,7 @@ spec:
         checksum/s3-secret: {{ printf "%s:%s" (include "backupDaemon.s3AccessKey" .) (include "backupDaemon.s3AccessSecret" .) | sha256sum }}
         {{- end }}
         {{- if .Values.tests.atpReport.enabled }}
-        checksum/atp-storage-secret: {{ include (print $.Template.BasePath "/atp-storage-secret.yaml") . | sha256sum }}
+        checksum/atp-storage-secret: {{ include (print $.Template.BasePath "/integration_tests/atp-storage-secret.yaml") . | sha256sum }}
         {{- end }}
     spec:
       {{- if .Values.tests.affinity }}

